examples/tf/cluster_demo.py

```diff
@@ -10,6 +10,7 @@
 
 from garage.baselines import LinearFeatureBaseline
 from garage.experiment import run_experiment
+from garage.runners import LocalRunner
```

Comment:
can you put this in experiment? i think it is most appropriate for now. we can always split it in the future.